packages/manager/modules/vps/src/veeam/vps-veeam.controller.js

```diff
@@ -56,7 +58,9 @@ export default class {
           (data) =>
             data.map((id) => ({
               id,
-              creationDate: moment(id).format('LLL'),
+              creationDate: moment(id)
```

Comment:
`id` doesn't sound like a variable parseable by moment, maybe rename to `date` or `time` or something that represents a value that moment can handle

Reply:
It was here before i made a changes.
And i had same reflection as you.
Updated.